src/workflow/OSWorkflow.cpp

```diff
@@ -34,16 +35,41 @@ OSWorkflow::OSWorkflow(const filesystem::path& oswPath, ScriptEngineInstance& ru
     workflowJSON(oswPath) {
 }
 
-OSWorkflow::OSWorkflow(WorkflowRunOptions t_runOptions, ScriptEngineInstance& ruby, ScriptEngineInstance& python)
+OSWorkflow::OSWorkflow(const WorkflowRunOptions& t_workflowRunOptions, ScriptEngineInstance& ruby, ScriptEngineInstance& python)
   :
 #if USE_RUBY_ENGINE
     rubyEngine(ruby),
 #endif
 #if USE_PYTHON_ENGINE
     pythonEngine(python),
 #endif
-    workflowJSON(t_runOptions.osw_path),
-    runOptions(std::move(t_runOptions)) {
+    workflowJSON(t_workflowRunOptions.osw_path),
+    m_no_simulation(t_workflowRunOptions.no_simulation),
+    m_post_process_only(t_workflowRunOptions.post_process_only),
+    m_show_stdout(t_workflowRunOptions.show_stdout),
+    m_add_timings(t_workflowRunOptions.add_timings),
+    m_style_stdout(t_workflowRunOptions.style_stdout) {
+
+  if (t_workflowRunOptions.runOptions.debug() || (workflowJSON.runOptions() && workflowJSON.runOptions()->debug())) {
+    fmt::print("Original workflowJSON={}\n", workflowJSON.string());
+    t_workflowRunOptions.debug_print();
+    fmt::print("m_no_simulation={}, m_post_process_only={}, m_show_stdout={}, m_add_timings={}, m_style_stdout={}", m_no_simulation,
+               m_post_process_only, m_show_stdout, m_add_timings, m_style_stdout);
+  }
+  auto runOpt_ = workflowJSON.runOptions();
+  if (!runOpt_) {
+    workflowJSON.setRunOptions(t_workflowRunOptions.runOptions);
+  } else {
+    auto ori_ftOptions = runOpt_->forwardTranslatorOptions();
+    workflowJSON.setRunOptions(t_workflowRunOptions.runOptions);
+    // user supplied CLI flags trump everything
+    ori_ftOptions.overrideValuesWith(t_workflowRunOptions.runOptions.forwardTranslatorOptions());
+    workflowJSON.runOptions()->setForwardTranslatorOptions(ori_ftOptions);
+  }
+
+  if (workflowJSON.runOptions()->debug()) {
+    fmt::print("workflowJSON={}\n", workflowJSON.string());
+  }
```

Comment:
Merge CLI provided run & ft options with the WorkflowJSON (workflow.osw) use for the simulation. CLI Flags take priority over the WorkflowJSON ones.